packages/grpc-js/src/channel.ts

```diff
@@ -234,16 +236,25 @@ export class ChannelImplementation implements Channel {
                       callStream
                     );
                   } catch (error) {
-                    callStream.cancelWithStatus(
-                      Status.UNAVAILABLE,
-                      'Failed to start call on picked subchannel'
-                    );
+                    /* An error here indicates thaat something when wrong with
```

Comment:
"thaat"

Reply:
Fixed it, plus two other errors in that comment.